share/smack/lib/smack-svcomp.c

```diff
@@ -4,11 +4,11 @@
 #include "smack-svcomp.h"
 
 void __VERIFIER_error(void) {
-  __VERIFIER_assert(0);
+  assert(0);
```

Comment:
Both `__VERIFIER_assert` and `assert` are identical. `__VERIFIER_assert` is preferred here.